templates/publisher/_beta-notification.html

```diff
@@ -0,0 +1,12 @@
+<section class="p-strip--light u-no-margin--top u-no-padding">
+  <div class="row p-beta-notification">
+    <div class="col-12">
+      <p>
+        <span class="p-muted-heading"><b>Beta</b></span>
+        Snapcraft.io is in Beta. Can't find something?
+        <a href="https://dashboard.snapcraft.io/snaps" class="p-link--external">Go back to the old version</a> or
+        <a href="https://github.com/canonical-websites/snapcraft.io" class="p-link--external">report an issue</a>.
```

Comment:
This link could be directly to `/issues` (as one in the footer).

Reply:
Good plan, thanks @bartaz